### ISSUE TITLE
Newsletter and Mass email issues

### ISSUE THREAD

**Issue by escservices** [tidusjar/Ombi#1380]
#### Ombi Version:

V 2.2.1

#### Update Branch:

Stable
#### Media Sever:

Plex

#### Media Server Version:

1.5.5.3634  

#### Operating System:

Windows 10

#### Mono Version (only if your not on windows)

(Place text here)

#### Applicable Logs (from `/logs/` directory or the Admin page):

```

No Logs

```

#### Problem Description:

When sending newsletter to admin, shows only 2 movies but they are current (I added about 7 movies this week though)  and for TV shows it shows a bunch but a random mix of content that was added recently and stuff I added last year.  And when you send a mass email to admin below the subject it says "Ombi Ombi Recently Added (Subject you entered) Powered by Ombi".  I thought it looked strange.  I know I'm asking for a lot but at some point an option to change the logo in the mass email would be cool, or even an option to replace it with text.  Thanks

#### Reproduction Steps:

Newsletter -> Send to admin
Mass Email -> Send to admin

**Comment by tidusjar** [comment]
Regarding it only send 2 movies, there must be some logs with the newsletter. Can you please include them.

**Comment by escservices** [comment]
Sorry for the wait, I was at work.  If you look at the time stamps it doesn't make sense, there doesn't appear to be any errors at the time of sending the email.  The most recent error is at 8:14:12PM but I sent the test email at 10PM.  

![errors](https://cloud.githubusercontent.com/assets/1287511/25059914/1a10749e-215e-11e7-94a0-e9c9ac84dbba.JPG)

`2017/04/14 20:14:08.842 Ombi.Core.Setup Info: Begin executing GetProfiles call to Radarr for quality profiles 
2017/04/14 20:14:12.232 Ombi.Api.ApiRequest Error: System.InvalidOperationException: There is an error in XML document (0, 0). ---> System.Xml.XmlException: Root element is missing.
   at System.Xml.XmlTextReaderImpl.Throw(Exception e)
   at System.Xml.XmlTextReaderImpl.ParseDocumentContent()
   at System.Xml.XmlTextReaderImpl.Read()
   at System.Xml.XmlTextReader.Read()
   at System.Xml.XmlReader.MoveToContent()
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderPlexLibraries.Read4_MediaContainer()
   --- End of inner exception stack trace ---
   at System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader, String encodingStyle, XmlDeserializationEvents events)
   at System.Xml.Serialization.XmlSerializer.Deserialize(TextReader textReader)
   at Ombi.Api.ApiRequest.DeserializeXml[T](String input) in C:\projects\requestplex\Ombi.Api\ApiRequest.cs:line 107 System.InvalidOperationException: There is an error in XML document (0, 0). ---> System.Xml.XmlException: Root element is missing.
   at System.Xml.XmlTextReaderImpl.Throw(Exception e)
   at System.Xml.XmlTextReaderImpl.ParseDocumentContent()
   at System.Xml.XmlTextReaderImpl.Read()
   at System.Xml.XmlTextReader.Read()
   at System.Xml.XmlReader.MoveToContent()
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderPlexLibraries.Read4_MediaContainer()
   --- End of inner exception stack trace ---
   at System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader, String encodingStyle, XmlDeserializationEvents events)
   at System.Xml.Serialization.XmlSerializer.Deserialize(TextReader textReader)
   at Ombi.Api.ApiRequest.DeserializeXml[T](String input) in C:\projects\requestplex\Ombi.Api\ApiRequest.cs:line 107
`

**Comment by dejected1** [comment]
similar issue on #1376. I posted there too. I'm having the same issue. 
From my post in #1376:
"i'm having the same problem. when i was on the stable branch the newsletter would only send out new movies and brand new tv shows that weren't already in plex. New tv episodes of existing tv shows already in plex never showed up. I then moved to EAP. Now it sends new tvs episodes for existing shows only if i manually run the plex scheduled jobs then manually force the email to be generated from scheduled jobs immediately afterwards. It doesn't detect any new tv show episodes on its own. The log does shows the newsletter being generated each time it's supposed to but it never finds any content. Movies do get added to the newsletter still correctly."
